e2e/tests/121-Portal/ViewPaProfilePage/ViewActivityOverviewMessageTab.spec.ts

```diff
@@ -0,0 +1,108 @@
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import {
+  amountVisa,
+  paymentNrVisa,
+  programIdVisa,
+  registrationVisa as registrationVisaDefault,
+} from '@121-service/seed-data/mock/visa-card.data';
+import NLRCProgram from '@121-service/seed-data/program/program-nlrc-ocw.json';
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { ProgramPhase } from '@121-service/src/shared/enum/program-phase.enum';
+import { StatusEnum } from '@121-service/src/shared/enum/status.enum';
+import {
+  changePhase,
+  doPayment,
+  waitForPaymentTransactionsToComplete,
+} from '@121-service/test/helpers/program.helper';
+import {
+  awaitChangePaStatus,
+  importRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { test } from '@playwright/test';
+import data from '../../../../interfaces/Portal/src/assets/i18n/en.json';
```

Comment:
Great that you are using this import now. I do think it can benefit from a more descriptive name so maybe something like:

```suggestion
import englishTranslations from '../../../../interfaces/Portal/src/assets/i18n/en.json';
```
or just translations

Reply:
Thank you for your comment and suggestions the more of those the better for the quality of the project.